share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -354,6 +354,11 @@ macro(rocm_setup_license HEADER_ONLY)
                 COMPONENT devel
             )
         endif()
+        file(READ ${CPACK_RESOURCE_FILE_LICENSE} LICENSEFILE)
+        string(FIND "${LICENSEFILE}" "MIT" MITCHECK)
+        if(NOT CPACK_RPM_PACKAGE_LICENSE AND ${MITCHECK} NOT EQUAL -1)
+                set(CPACK_RPM_PACKAGE_LICENSE "MIT")
+        fi()
```

Comment:
Closing if statements in CMake uses `endif`, not `fi`.

Reply:
Thanks, I missed this. As per my above comment:

Do you have 2 components you suggest that I can test as an example? (one with MIT, one without)